### ISSUE TITLE
since two or three days, the metadata is not show

### ISSUE THREAD

**Issue by deegosu** [ccd0/4chan-x#3286]
Every time you try to see the "title" in a post with a webm, it says "not found" even though there is something written in the metadata of the webm... 
![32](https://user-images.githubusercontent.com/110716980/183240672-cd65fc13-523f-4810-bbfa-4aaafe0c750f.PNG)

I have tested it with some webms made by me recently, I also tested with old webms that used to show their title and it doesn't find it either
 At first I thought it was only in VP9 webms but it's happening with VP8 too, since two days at least

**Comment by deegosu** [comment]
I'm using the chrome extension for microsoft Edge browser btw

**Comment by saxamaphone69** [comment]
I have a feeling `Metadata` just flat out stopped working? The title seems to be stripped either on upload/save.
| webm that I created with a title: | same webm when saved: |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/877480/235339815-a3031616-1eb6-4630-b0c7-514b317e61dd.png) | ![image](https://user-images.githubusercontent.com/877480/235339853-38525b9a-ae48-49db-af09-ffdb010af716.png) |

**Comment by bakugo** [comment]
Yeah, 4chan started stripping as much metadata as possible from uploads a year or so ago because of embed scripts.
